frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -275,14 +277,14 @@ export const HorizontalNav = React.memo((props: HorizontalNavProps) => {
 
   const routes = pages.map((p) => {
     const path = `${props.match.path}/${p.path || p.href}`;
-    const render = (params) => {
+    const render = (params: RouteComponentProps) => {
       return (
         <ErrorBoundary FallbackComponent={ErrorBoundaryFallback}>
           <p.component
             {...componentProps}
             {...extraResources}
             {...p.pageData}
-            params={params}
+            {...params}
```

Comment:
Won't this be a big change to existing pages because now the params are spread instead of in a prop named `params` ?

Reply:
I checked and none of the usage is using `params`.